app/javascript/articles/components/CommentsCount.jsx

```diff
@@ -28,7 +30,7 @@ export const CommentsCount = ({ count, articlePath }) => {
           {count}
           <span className="hidden s:inline">
             &nbsp;
-            {`${count > 1 ? 'comments' : 'comment'}`}
+            {`${count > 1 ? `${locale('core.comment')}s` : locale('core.comment') }`}
```

Comment:
Please do not hardcode one-or-many type branch into i18n logic. There are way more languages not working this way than those are. Take `count` as argument. See for example: https://phrase.com/blog/posts/rails-i18n-guide/

Plus, plural = singular + "s" is only practically feasible for English and French...

Reply:
I think I brought up things like this in the above comments, also regarding Date formats, etc. All these things will be fixed/improved in due time, think of this as the absolute baseline.

Reply:
I thought I have looked though past comments on this conversation page, but sorry if I missed your comment.

Reply:
You know, maybe I shared that on Slack or elsewhere not here but the most important thing is that we agree on handling things like pluralization, date formats, etc. properly. Please continue pointing out these things in future PRs when you notice them.